web/src/features/auth/slices/authenticationSlice.test.ts

```diff
@@ -1,18 +1,105 @@
-import { decodeRoles } from 'features/auth/slices/authenticationSlice'
+import authReducer, {
+  decodeRoles,
+  initialState,
+  authenticateStart,
+  authenticateFinished,
+  authenticateError,
+  refreshTokenFinished
+} from 'features/auth/slices/authenticationSlice'
 
 describe('authenticationSlice', () => {
+  const tokenWithRoles =
```

Comment:
I don't mind this in the tests - but feels wrong having something like this in web/src/features/auth/AuthWrapper.tsx

Reply:
Not sure where to put the one in `AuthWrapper.tsx` besides maybe `web/src/features/auth/roles.ts` only because it's a set of constants in there to do with roles/auth.